tests/test_replication.py

```diff
@@ -60,13 +61,13 @@ def test_incoming_replication(self):
 
         # Configure the Sydent to impersonate. We need to use "fake.server" as the
         # server's name because that's the name the recipient Sydent has for it. On top
-        # of that, the replication servlet expects a TLS certificate in the request so it
-        # can extract a common name and figure out which peer sent it from its common
+        # of that, the replication servlet expects a TLS certificate in the request so
+        # it can extract a common name and figure out which peer sent it from its common
         # name. The common name of the certificate we use for tests is fake.server.
         config = {
             "general": {"server.name": "fake.server"},
             "crypto": {
-                "ed25519.signingkey": "ed25519 0 b29eXMMAYCFvFEtq9mLI42aivMtcg4Hl0wK89a+Vb6c"
+                "ed25519.signingkey": "ed25519 0 b29eXMMAYCFvFEtq9mLI42aivMtcg4Hl0wK89a+Vb6c"  # noqa: E501
```

Comment:
It might make more sense to globally disable this [like we do for Synapse](https://github.com/matrix-org/synapse/blob/ed53bf314fee25d79d349beae409caf81a2d677f/setup.cfg#L13-L21) (for the same reasoning, that black handles it for us).